src/styles/blocks/is-active.scss

```diff
@@ -0,0 +1,13 @@
+.is-active.menu__link {
+  color: $main-header-color;
+
+  &::after {
+    content: ' ';
+    position: absolute;
```

Comment:
The use of 'position: absolute;' requires that the parent element has a 'position' property set to 'relative', 'absolute', or 'fixed' to ensure the pseudo-element is positioned correctly. Ensure that the parent of '.is-active.menu__link' is positioned appropriately.